docs/operations.md

```diff
@@ -1,13 +1,139 @@
+      # https://backstage.io/docs/getting-started/configuration#setting-up-a-github-integration
+      # token: ${GITHUB_TOKEN}
+      apps:
+        - $include: github-app-backstage-bits-dev-credentials.yaml
+
+
+techdocs:
+  builder: 'local' # Alternatives - 'external'
+  generator:
+    runIn: 'docker' # Alternatives - 'local'
+  publisher:
+    type: 'local' # Alternatives - 'googleGcs' or 'awsS3'. Read documentation for using alternatives.
+
+catalog:
+  locations:
+      # Local example template
+    - type: file
+      target: ../../templates/scaffolder/pr-with-catalog-entry/template.yaml
+      rules:
+        - allow: [Template]
+
+    - type: file
+      target: ../../templates/scaffolder/terraform-control-module/template.yaml
+      rules:
+        - allow: [Template]
+
+    - type: file
+      target: ../../templates/scaffolder/terraform-with-terragrunt/template.yaml
+      rules:
+        - allow: [Template]
+
+    - type: file
+      target: ../../templates/scaffolder/terraform-storage-transfer/template.yaml
+      rules:
+        - allow: [Template]
+
+    - type: file
+      target: ../../examples/entities.yaml
+
+    # Local example template
+    - type: file
+      target: ../../examples/template/template.yaml
+      rules:
+        - allow: [Template]
+
+    # Local example PagerDuty template
+    - type: file
+      target: ../../examples/template/pagerduty.yaml
+      rules:
+        - allow: [Template]
+
+    # Local example organizational data
+    - type: file
+      target: ../../examples/org.yaml
+      rules:
+        - allow: [User, Group]
+
+```
+
+
 ## Local Development
 
-In order to run backstage locally, you need to set up a few things.
+In order to run backstage locally, you need to set up a few things after cloning the repo.
+
+Node and yarn will need to be installed. For the correct version of node, you can use `nvm` to manage your node versions. This will pick up the correct version from the .nvmrc file in the repo.
+
+
+#### Local Secrets
+
+To run Backstage locally, you need to set up a number of secrets locally. These secrets are stored in 1Password, and can be found in the `Backstage` vault. The secrets can be manually sourced, or you can use the `1password-cli` to source them. To use the `1password-cli`, you need to install it and authenticate with your 1Password account. You can find instructions on how to do this [here](https://support.1password.com/command-line-getting-started/).
+
+This repo uses [direnv](https://direnv.net/) to manage environment variables. You can install direnv by running `brew install direnv` on MacOS. Once you have direnv installed, you can use the `.envrc` files in the repo to source the secrets. You can also manually source the secrets by running the following commands from the app root (i.e the backstage directory in the repo): `source op.bash`
+
+This is a sample `op.bash` file that you can use to source the secrets:
```

Comment:
I think we can just reference the actual file that exists to avoid documentation drift

Reply:
yes, and it may be more trouble than it's worth. Putting a standard `[link](../backstage/op.bash)` won't work, because techdocs aren't served from the repo, they get rendared and then pushed to a GCS bucket - basicly the link works, but the op.back file doesn't exist in the directory with the rendered pages. I could put a full url link in, but that seems kind of meh, since then it points the a particular version of the file in main. I could remove the sample completly and just point the user toward the file without a link I suppose...